<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddContactDlg</class>
 <widget class="QDialog" name="AddContactDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddContactDlg</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QLineEdit" name="phoneNumLineEdit"/>
   </item>
   <item row="2" column="1">
    <widget class="QPlainTextEdit" name="addressTextEdit"/>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="nameLineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Address:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Phone Number:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>nameLineEdit</tabstop>
  <tabstop>phoneNumLineEdit</tabstop>
  <tabstop>addressTextEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
